### ISSUE TITLE
[Umbrella Issue] Create list of tests for promotion that hit unconformant stable endpoints

### ISSUE THREAD

**Issue by devaips** [kubernetes/kubernetes#78747]
> This issue is kept in sync with [this external document.](https://hackmd.io/@rQlNUtY8RpqEOCyNLvoobg/S1UNYufbH)

Filtering through auditlog data [using APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=level-stable) gets us a nice list of Stable API endpoints which are currently tested but not part of the Conformance suite. 

Here is the list of tests which hit those endpoints, excluding Conformance tests, and excluding Stable/Core as those have already been tracked and promoted in https://github.com/kubernetes/kubernetes/issues/78748

| # | Endpoint | Tests | Kind | Comments |  
| -  | - | - | - | - |  
| 1 | replaceAppsV1NamespacedReplicaSet | [sig-apps] ReplicaSet should surface a failure condition on a common issue like exceeded quota | ReplicaSet | PR: https://github.com/kubernetes/kubernetes/pull/80148 |
| 2 | ~~deleteAppsV1NamespacedReplicaSet~~ | ~~[sig-api-machinery] ResourceQuota should create a ResourceQuota and capture the life of a replica set.~~ | ReplicaSet | Test does not qualify for promotion. It uses a PVC. |
| 3 | replaceAppsV1NamespacedStatefulSetScale | [sig-apps] StatefulSet [k8s.io] Basic StatefulSet functionality [StatefulSetBasic] should have a working scale subresource | Scale | PR: https://github.com/kubernetes/kubernetes/pull/80150 |
| 4 | readAppsV1NamespacedStatefulSetScale | [sig-apps] StatefulSet [k8s.io] Basic StatefulSet functionality [StatefulSetBasic] should have a working scale subresource | Scale | PR: https://github.com/kubernetes/kubernetes/pull/80150 |
| 5 | createAuthorizationV1SelfSubjectAccessReview | [sig-api-machinery] Servers with support for Table transformation should return a 406 for a backend which does not implement metadata | SelfSubjectAccessReview | PR: https://github.com/kubernetes/kubernetes/pull/80153 |
| 6 | createAutoscalingV1NamespacedHorizontalPodAutoscaler | [sig-autoscaling] [HPA] Horizontal pod autoscaling (scale resource: CPU) [sig-autoscaling] ReplicationController light Should scale from 1 pod to 2 pods | HorizontalPodAutoscaler | PR: https://github.com/kubernetes/kubernetes/pull/79954 |
| 7 | deleteAutoscalingV1NamespacedHorizontalPodAutoscaler | Sequence For [sig-autoscaling] [HPA] Horizontal pod autoscaling (scale resource: CPU) [sig-autoscaling] ReplicationController light Should scale from 1 pod to 2 pods | HorizontalPodAutoscaler | PR: https://github.com/kubernetes/kubernetes/pull/79954 |
| 8 | logFileListHandler | [ ~~sig-network] Networking should provide unchanging, static URL paths for kubernetes api services~~ | logs | Does not meet Conformance Requirements: [Pre-Test Assertion](https://github.com/kubernetes/kubernetes/blob/08a36f6e550a6c0fbdbca5e58294502adf27566e/test/e2e/network/networking.go#L35) requires access to Public Internet, however it looks like this test [does not actually need](https://github.com/kubernetes/kubernetes/blob/08a36f6e550a6c0fbdbca5e58294502adf27566e/test/e2e/network/networking.go#L62) that. |
| 9 | createSchedulingV1PriorityClass | [sig-scheduling] PreemptionExecutionPath runs ReplicaSets to verify preemption running path | PriorityClass | PR: [#79959](https://github.com/kubernetes/kubernetes/pull/79959) |
| 10 | deleteSchedulingV1PriorityClass | [sig-scheduling] PreemptionExecutionPath runs ReplicaSets to verify preemption running path  | PriorityClass | PR: [#79959](https://github.com/kubernetes/kubernetes/pull/79959) |
| 11 | listStorageV1StorageClass | Many - [See APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-storage-listStorageV1StorageClass) | StorageClass | We need a generic storage solution before we can promote MOST if not all of these. |
| 12 | readStorageV1StorageClass | Many - See [APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-storage-readStorageV1StorageClass) | StorageClass | Closed PR: https://github.com/kubernetes/kubernetes/pull/79962 - We need a generic storage solution before we can promote MOST if not all of these.  |
| 13 | deleteAppsV1NamespacedDaemonSet | Many - See [APISNoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-apps-deleteAppsV1NamespacedDaemonSet) | DaemonSet | |
| 14 | createAppsV1NamespacedDaemonSet | Many - See [APISNoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-apps-createAppsV1NamespacedDaemonSet) | DaemonSet | |
| 15 | deleteRbacAuthorizationV1NamespacedRole | Many - See [APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-rbacAuthorization-deleteRbacAuthorizationV1NamespacedRole) | rbacAuthorization | |
| 16 | createRbacAuthorizationV1NamespacedRole | Many - See [APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-rbacAuthorization-createRbacAuthorizationV1NamespacedRole) | rbacAuthorization | |
| 17 | deleteStorageV1StorageClass | Many - See [APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-storage-deleteStorageV1StorageClass) | StorageClass | We need a generic storage solution before we can promote MOST if not all of these. |
| 18 | createStorageV1StorageClass | Many - See [APISnoop](https://apisnoop.cncf.io/?useragents=%5B%5Ee2e.test%5D&showUntested=false&showTested=true&showConformanceTested=false&zoomed=operationId-stable-storage-createStorageV1StorageClass) | StorageClass | We need a generic storage solution before we can promote MOST if not all of these. |

/area test
/area conformance

**Comment by devaips** [comment]
As soon as https://github.com/kubernetes/kubernetes/pull/80148 is closed/merged then we can close this issue.

**Comment by spiffxp** [comment]
/priority important-soon
ref: https://docs.google.com/document/d/1W31nXh9RYAb_VaYkwuPLd1hFxuRX3iU0DmaQ4lkCsX8/edit#bookmark=id.xra9sskijfk7

**Comment by devaips** [comment]
Issues which were not merged are superceeded by https://github.com/kubernetes/kubernetes/pull/81574
